### ISSUE TITLE
how to plot recall-precision curve and print mAP on the terminal along with the loss

### ISSUE THREAD

**Issue by onurbarut** [thtrieu/darkflow#532]
Hey all,

Can anyone help me to plot recall-precision curve and print mAP on the terminal along with the loss?

Also has anyone applied validation flow after particular number of steps? I want to see the loss of train and validation sets in the same plot. Any help?

Thanks.

**Comment by yyuuliang** [comment]
hi
I recently finished a small project which can plot AP against Recall and the mAP. But the code is not 100% compatible with the Darkflow code here. If you want to do it, you can check my code here
https://github.com/yyuuliang/darkflow/tree/udacity18

**Comment by yyuuliang** [comment]
try this, it is a small project I worked on last month.
https://github.com/yyuuliang/darkflow/tree/udacity18
